src/contracts/PreAMMBatcher.sol

```diff
@@ -102,10 +104,25 @@ contract PreAMMBatcher {
         return Fraction(f.denominator, f.numerator);
     }
 
+    function orderIsCurrentlyValid(Order memory order)
+        private
+        view
+        returns (bool)
+    {
+        // Zero velidFrom is a sentinel value signaling that the order has unlimited validity.
+        // validUntil can then be set to zero to save on gas costs.
+        // solhint-disable not-rely-on-time
+        return
+            order.validFrom == 0 ||
+            ((order.validUntil >= block.timestamp) &&
+                (order.validFrom <= block.timestamp));
+        // solhint-enable not-rely-on-time
+    }
+
     function orderChecks(
         Order[] memory sellOrderToken0,
         Order[] memory sellOrderToken1
-    ) internal pure {
+    ) internal view {
```

Comment:
Which aspect of this required the change to `view` from `pure`?

Reply:
Getting the block timestamp requires methods to be "view" AFAIK.